homeassistant/components/zha/core/channels/closures.py

```diff
@@ -23,6 +24,27 @@ async def async_update(self):
                 f"{self.unique_id}_{SIGNAL_ATTR_UPDATED}", 0, "lock_state", result
             )
 
+    @callback
+    def cluster_command(self, tsn, command_id, args):
+        """Handle a cluster command received on this cluster."""
+
+        # Operational Event
+        if (
+            command_id == 32
+            and self._cluster.client_commands is not None
+            and self._cluster.client_commands.get(command_id) is not None
+            and isinstance(args[0], closures.DoorLock.OperationEventSource)
+            and isinstance(args[1], closures.DoorLock.OperationEvent)
```

Comment:
This is a tad redundant. If the `command_id == 32` you are assured for the args to be a list of `[                OperationEventSource, OperationEvent, t.uint16_t, t.uint8_t, t.LocalTime, t.CharacterString, ]`

alternatively you could do
```
    command_name = self._cluster.client_commands.get(command_id, [command_id])[0]
    if command_name == "operation_event_notification":
        self.zha_send_event(...
```

this also improves readability, as now I don't have to look up what command id 32 means.